neurom/core/tests/test_iter.py

```diff
@@ -52,7 +53,7 @@
 POP = core.Population(NEURONS, name='foo')
 
 # To make transition to morphio easier
-MORPHIO_OFFSET = 0
+MORPHIO_OFFSET = 2
```

Comment:
this is just of the test, right? Why do you need to put =2 ? Is this relaated to the fact that the soma is not part of .sections anymore?

Reply:
This is just for test so no need to update data in this test. MorphIO does not count soma sections as sections anymore. Hence this offset.

Reply:
Why is it 2? I would have expected 1? (naive question)

Reply:
This variable exist, so we can keep the existing test data unchanged. It is 2 because before there was a soma with 2 sections. NeuroM v1 includes soma sections. NeuroM v2 does not include soma sections => those 2 soma sections from before are out.